hypertrace-trace-enricher/hypertrace-trace-enricher/src/main/resources/configs/common/application.conf

```diff
@@ -105,6 +105,11 @@ enricher {
     class = "org.hypertrace.traceenricher.enrichment.enrichers.ExitCallsEnricher"
     dependencies = ["DefaultServiceEntityEnricher", "ApiBoundaryTypeAttributeEnricher", "BackendEntityEnricher"]
   }
+
+  TraceStatsEnricher {
+    class = "org.hypertrace.traceenricher.enrichment.enrichers.TraceStatsEnrichere"
+    dependencies = ["DefaultServiceEntityEnricher", "ApiBoundaryTypeAttributeEnricher", "BackendEntityEnricher"]
```

Comment:
@kotharironak copied dependencies from ExitCallEnricher, although not sure if all needed here

Reply:
This will be fine as it indirectly covers deps on `EndpointEnricher`. However, we can be explicit and go with
```
dependencies = ["EndpointEnricher"]
```

Reply:
Thanks @kotharironak. Will make it explicit.